### PR TITLE
[Monorepo] Add Cypress and Eslint Nx plugins

### PR DESCRIPTION
**Pull request by icaparros-at-wiris**
## Description

This PR adds Cypress and Eslint Nx plugins to the monorepo project. This plugins allows us to pass Tests and Linting on all workspace packages or an specific one using the nx commands.

## Steps to reproduce

To run tests from specific package:
1. Run `yarn` on the project root.
2. Run `nx build <package-name>` to build all packages.
3. Run `nx start <demo-name>` to start all demos.
4. Run `nx test <package-name>` to run all packages tests.

To lint specific package:
1. Run `yarn` on the project root.
2. Run `nx lint <package-name>` to run eslint on the package.

To lint all packages in parallel:
1. Run `yarn` on the project root.
2. Run `nx run-many --target=lint --all --parallel` to run eslint on all packages.

---

[#taskid 25401](https://wiris.kanbanize.com/ctrl_board/2/cards/25401/details/)

### CODE REVIEW

**Review by carla-at-wiris — APPROVED** [review]
Everything looks fine for me! I'll check again once we add the demos to this branch.

File: docs/development/testing/README.md [hunk 1]
```diff
@@ -31,16 +31,24 @@ $ xhost local:root
...
 
-// TODO
\ No newline at end of file
+You can run all tests for a specific demo with the `nx test <package>` command.
+
+Before running the tests you will need to build the package and start a demo. For example to run all tests on the `ckeditor5` demo run:
+
+```
+$ nx build ckeditor5
+$ nx start ckeditor5
```
Comment (xripoll-at-wiris) [inline docs/development/testing/README.md:52]: Watch out; the demo is `html-ckeditor5`, not just `ckeditor5`.

**Comment by xripoll-at-wiris** [comment]
I understand that some lints and tests are expected to fail, right? I am getting failed tests with `html-tinymce5` and failed lints on 7 (all?) packages.

**Comment by icaparros-at-wiris** [comment]
> I understand that some lints and tests are expected to fail, right? I am getting failed tests with `html-tinymce5` and failed lints on 7 (all?) packages.

All tests that exist are designed for the generic demo, for the rest demos just the first test will pass sucessfully then the second one will fail and the rest will be skiped.
> Will be skiped as the parameter `FAIL_FAST_ENABLED` is set to `true` on the `cypress.json` file from the root. 

In the future we will have to adapt the demos and tests to be executable for all the editors adding the same id or class to all the elements that we want to interact.

I think it may be a good idea to leave this configuration even if the tests fail, because we have plans to update and add tests in the near future.

**Review by carla-at-wiris — APPROVED** [review]

**Comment by xripoll-at-wiris** [comment]
> All tests that exist are designed for the generic demo, for the rest demos just the first test will pass sucessfully then the second one will fail and the rest will be skiped.

> About linting, I have configured the linter to lint all packages including the demos. 7 packages have some lint error. Maybe we can create a task to fix all linting errors.

Okay, then everything works as expected.

Indeed, fixing these tests/lints is not part of this task. Eventually we will have to fix them but for now we are good.

**Review by xripoll-at-wiris — APPROVED** [review]

**Pull request merged**